### ISSUE TITLE
Pretrained weight only for English?

### ISSUE THREAD

**Issue by abghaky** [csxmli2016/textbsr#10]
Thank you for the great project.

I want to use this model only for super-resolution of English text and numbers. In my tests, whenever the text quality is low, some of the letters are interpreted as Chinese characters, and I want to avoid that. 

By any chance, do you have any pre-trained weights only for English? If not, can you guide on how to do the training? Also, should I train the model from scratch, or just finetune it?

**Comment by csxmli2016** [comment]
> Thank you for the great project.
> 
> I want to use this model only for super-resolution of English text and numbers. In my tests, whenever the text quality is low, some of the letters are interpreted as Chinese characters, and I want to avoid that.
> 
> By any chance, do you have any pre-trained weights only for English? If not, can you guide on how to do the training? Also, should I train the model from scratch, or just finetune it?

Thanks for your interest. This model is trained using Chinese text images. If you want to focus on English and numbers, you'd better finetune this model on them. This model is trained using [BSRGAN](https://github.com/cszn/BSRGAN). The input is 64\*128 and the output is 256\*512. You can also use our [synthetic pipeline of training data from MARCONet](https://github.com/csxmli2016/MARCONet/blob/main/Train/syndata_demo.py). Just modify the corpus (Line 294, get_text()) to get English or Numbers or their combination. Then you can crop the image based on the random x-axis location.
